internal/model/command.go

```diff
@@ -349,6 +353,20 @@ func runWithVersion(cmd *cobra.Command, artifactArch, desiredVersion string) err
 		return fmt.Errorf("failed to run with version %s: %w", desiredVersion, err)
 	}
 
+	// If the workflow succeeded, make the used version the default
+	if promote && !env.IsGithubAction() && !env.IsLocalDev() {
+		currentExecPath, err := os.Executable()
```

Comment:
I suppose there is a valid reason for us to prompt and only do this when we are in interactive mode and we know we can. Idk what do you think

Reply:
I tried that, and I'm open to it, but it just felt a bit tedious. Like, if the CLI just ran using that version, why would I _not_ want to install it (that's what im imagining customers saying/thinking)